conformance-checking/src/test/java/org/archcnl/conformancechecking/impl/ConformanceCheckImplTest.java

```diff
@@ -77,35 +79,23 @@ public void testCoarse() throws FileNotFoundException {
         actual.read(new FileReader(outputPath), "");
 
         // workaround: remove the checking date
+        DatatypeProperty hasCheckingDate =
+                ConformanceCheckOntologyClassesAndProperties.get(
+                        ConformanceCheckDatatypeProperties.hasCheckingDate, expected);
         expected.getProperty(
                         ConformanceCheckOntologyClassesAndProperties.getConformanceCheckIndividual(
                                 expected),
-                        ConformanceCheckOntologyClassesAndProperties.getDateProperty(expected))
+                        hasCheckingDate)
                 .remove();
+        hasCheckingDate =
+                ConformanceCheckOntologyClassesAndProperties.get(
+                        ConformanceCheckDatatypeProperties.hasCheckingDate, actual);
         actual.getProperty(
                         ConformanceCheckOntologyClassesAndProperties.getConformanceCheckIndividual(
                                 actual),
-                        ConformanceCheckOntologyClassesAndProperties.getDateProperty(actual))
+                        hasCheckingDate)
                 .remove();
 
-        System.out.println("IN EXPECTED BUT NOT IN ACTUAL:");
-        expected.listStatements()
-                .forEachRemaining(
-                        (s) -> {
-                            if (!actual.contains(s)) {
-                                System.out.println(s.toString());
-                            }
-                        });
-
-        System.out.println("IN ACTUAL BUT NOT IN EXPECTED:");
-        actual.listStatements()
-                .forEachRemaining(
-                        (s) -> {
-                            if (!expected.contains(s)) {
-                                System.out.println(s.toString());
-                            }
-                        });
-
```

Comment:
This looks like a debugging relic so we should be fine removing it although this will make future debugging harder if the error arises once again.